COPYRIGHT.txt

```diff
@@ -27,32 +27,33 @@
 #   License: Zlib
 # The exact copyright for each file in that library *may* differ, and some
 # files or code snippets might be distributed under other compatible licenses
-# (e.g. a public domain dedication), but as far as Godot Engine is concerned
+# (e.g. a public domain dedication), but as far as Blazium Engine is concerned
 # the library is considered as a whole under the Zlib license.
 #
 # Note: When linking dynamically against thirdparty libraries instead of
-# building them into the Godot binary, you may remove the corresponding
+# building them into the Blazium binary, you may remove the corresponding
 # license details from this file.
 
 -----------------------------------------------------------------------
 
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
-Upstream-Name: Godot Engine
-Upstream-Contact: Rémi Verschelde <contact@godotengine.org>
-Source: https://github.com/godotengine/godot
+Upstream-Name: Blazium Engine
+Upstream-Contact: https://chat.blazium.app
+Source: https://github.com/blazium-engine/blazium
 
 Files: *
-Comment: Godot Engine
-Copyright: 2014-present, Godot Engine contributors
+Comment: Blazium Engine
+Copyright: 2024-present, Blazium Engine contributors
+ 2014-present, Godot Engine contributors
  2007-2014, Juan Linietsky, Ariel Manzur
 License: Expat
 
 Files: ./icon.png
  ./icon.svg
  ./logo.png
  ./logo.svg
-Comment: Godot Engine logo
-Copyright: 2017, Andrea Calabró
+Comment: Blazium Engine logo
+Copyright: 2024, [LOGE (on X/Twitter)
```

Comment:
what is LOGE

Reply:
If I recall correctly this is what someone told me was the user name of the person who made the logo, if incorrect please let me know the information I need to put here :)

Reply:
ok, didn't know if it was meta data or something, all good.

Reply:
Honestly, I'm not 100% sure this is correct, as I don't recall the specific conversation at this point I got LOGE from, if there is a way to verify this I would appreciate it.

Reply:
Verified copyright information with copyright holders, see https://discord.com/channels/1294004245489782896/1294475944177831966/1299382663005732935 for more information